tools/staramr/staramr_search.xml

```diff
@@ -1,10 +1,14 @@
 <tool id="staramr_search" name="staramr" version="@VERSION@+galaxy0">
```

Comment:
I think you want to specify the version of Galaxy (`profile=""`) the tool was designed for somewhere here.

https://docs.galaxyproject.org/en/latest/dev/schema.html#examples